ThemeConfig.swift

```diff
@@ -83,4 +83,14 @@ struct ColorPallete {
         green: 2.0 / 255.0,
         blue: 9.0 / 255.0
     )
+    
+    static let navLow = Color(
+        red: 199.0 / 255.0,
+        green: 199.0 / 255.0,
+        blue: 199.0 / 255.0
+    );
+    
+    static let greenGradientStart = Color(red: 158 / 255, green: 232 / 255, blue: 0) // #9EE800
+    
+    static let greenGradientEnd = Color(red: 4 / 255, green: 233 / 255, blue: 0) // #04DF00
```

Comment:
One pain-point for this Figma project right now is there's like 10+ unique shades of greens used.  To make sure things don't get too out of hand, I chose like ~6 (see the rest of the ThemeConfig.swift file), and am trying to not add more greens for one off items, like this button. I would suggest trying to see if any of the other colors work. If really none of them work, we can add these new ones in. And if we do add them in, maybe we could try to fit them into the current naming scheme? (midGreen, etc)

Reply:
Do you mind if I just define them in the function that uses them since they are one off?

Reply:
Probably best to put the new ones in this file. Probably just need a new one for that yellow-ish color.

It's my B, since I basically just eyeball the color for every new element. I don't love the way that Figma manages standard colors, so I don't use it that much.

Working on a new revision of the home page design right now, and will make sure to solidify the color scheme there, so this eventually gets cleaned up & standardized.